config/webpack.dev.js

```diff
@@ -69,6 +69,22 @@ if (devMode === 'local') {
   plugins.push(
     new webpack.NormalModuleReplacementPlugin(/client_factory.ts/, 'client_factory.mock.ts')
   );
+  plugins.push(
+    new webpack.NormalModuleReplacementPlugin(/plan\/duck\/operations\.ts/, res => {
```

Comment:
This is something we'd like to avoid.

Would like to keep the mock interface at the client level.